portfolio/src/main/webapp/electronics.html

```diff
@@ -0,0 +1,91 @@
+            coords="435, 165, 441, 244, 554, 256, 639, 160, 595, 88, 533, 170" 
+            shape="poly">
+        <area alt="Oscilloscope" onclick="moreInfo('oscilloscope')" 
+            coords="476, 166, 531, 164, 596, 86, 596, 49, 473, 28" 
+            shape="poly">
+        <area alt="STM32 Midi Project" onclick="moreInfo('stm32MidiProject')" 
+            coords="207, 184, 202, 211, 407, 233, 408, 216, 311, 195, 248, 171" 
+            shape="poly">
+        <area alt="Assorted Lengths of Wire" onclick="moreInfo('wireDrawer')" 
+            coords="398, 260, 393, 310, 467, 317, 563, 318, 573, 280" 
+            shape="poly">
+        <area alt="Sanding Mask" onclick="moreInfo('mask')" 
+            coords="360, 141, 355, 157, 401, 161, 402, 145, 381, 131" 
+            shape="poly">
+        <area alt="Development Boards" onclick="moreInfo('devBoards')" 
+            coords="233, 79, 215, 107, 217, 119, 266, 110, 273, 88" 
+            shape="poly">
+        <area alt="Schematics" onclick="moreInfo('schematics')" 
+            coords="213, 138, 227, 163, 277, 167, 356, 149, 338, 122, 289, 116" 
+            shape="poly">
+        <area alt="Soldering Equipment" onclick="moreInfo('soldering')" 
+            coords=" 402, 101, 394, 124, 443, 127, 444, 100" 
+            shape="poly">
+        <area alt="Various Tapes" onclick="moreInfo('tape')" 
+            coords="448, 111, 442, 145, 434, 157, 455, 164, 472, 161, 467, 147, 461, 112" 
+            shape="poly">
+        <area alt="Canned Air" onclick="moreInfo('cannedAir')"
+          coords="389, 44, 383, 70, 376, 110, 389, 123, 399, 69, 397, 59, 384, 61" 
+          shape="poly">
+        <area alt="Flush Cutters" onclick="moreInfo('snips')" 
+            coords="301, 167, 309, 190, 341, 178, 325, 168" 
+            shape="poly">
+        <area alt="LEDs and transistors" onclick="moreInfo('LedsTransistors')" 
+            coords="277, 76, 270, 105, 306, 114, 314, 75" 
+            shape="poly">
+        <area alt="Breadboards" onclick="moreInfo('breadboards')" 
+            coords="317, 89, 310, 116, 325, 115, 331, 91" 
+            shape="poly">
+        <area alt="Resistors" onclick="moreInfo('resistors')" 
+            coords="170, 49, 179, 96, 201, 99, 193, 50" 
+            shape="poly">
+        <area alt="Post-it Notes" onclick="moreInfo('postItNotes')" 
+            coords="350, 82, 350, 120, 371, 118, 369, 84" 
+            shape="poly">
+        <area alt="Halogen Light" onclick="moreInfo('light')" 
+            coords="19, 177, 25, 208, 33, 211, 53, 204, 67, 184, 59, 163, 37, 162" 
+            shape="poly">
+      </map>
+      <div id="electronics-info">
+        <p>Click on an item in the image for more information here.</p>
+      </div>
+      <p>If you want to learn more about my electronics, or just electronics 
+        in general, check out the projects on my projects page.
+      </p>
+    </div>
+  </body>
+</html>
```

Comment:
nit: Add a newline at the end of the file (I'm not sure about Cloud Shell specifically but most editors have an option to just do that automatically)